addons/stock/models/stock_picking.py

```diff
@@ -660,15 +660,20 @@ def _default_picking_type_id(self):
     is_date_editable = fields.Boolean(
         'Is Scheduled Date Editable', compute='_compute_is_date_editable')
 
+    weight = fields.Float(compute='_compute_weight', digits='Stock Weight', store=True, help="Total weight of the products in the picking.", compute_sudo=True)
```

Comment:
[It's true by default for stored fields.](https://www.odoo.com/documentation/master/developer/reference/backend/orm.html#:~:text=that%20contains%20them.-,compute_sudo%20(bool),-%E2%80%93%20whether%20the%20field)
```suggestion
    weight = fields.Float(compute='_compute_weight', digits='Stock Weight', store=True, help="Total weight of the products in the picking.")
```